balsam/config/defaults/olcf_summit/settings.yml

```
@@ -0,0 +1,106 @@
+# Default setup for Summit cluster @ OLCF
+
+site_id: {{ site_id }}  # Do not change
+
+# Log file behavior for launcher and site services
+logging:
+    level: DEBUG # Increasing verbosity: ERROR, WARNING, INFO, DEBUG
+    format: '%(asctime)s|%(process)d|%(thread)d|%(levelname)8s|%(name)s:%(lineno)s]
+        %(message)s'
+    datefmt: '%d-%b-%Y %H:%M:%S'
+    buffer_num_records: 1024  # Flush logs after this many records emitted
+    flush_period: 30  # Flush logs after this many seconds since last flush
+
+# Use filter_tags to limit what jobs the Site will process.
+# Launchers submitted from the command line will ignore this setting 
+# and use the --filter-tags specified as command line arguments.
+# For example:
+#   filter_tags:
+#     system: H2O
+#     workflow: test
+filter_tags: {}
+
+
+# Launcher settings
+launcher:
+    idle_ttl_sec: 30  # Shutdown after this many seconds of nothing running
+    delay_sec: 1  # Process polling interval
+    error_tail_num_lines: 10 # Number of lines from tail-end of job output to store
+    max_concurrent_mpiruns: 990 # Maximum number of runs in a single MPI-mode launcher
+    
+    # Launcher platform interfaces:
+    compute_node: balsam.platform.compute_node.SummitNode
+    mpi_app_launcher: balsam.platform.app_run.SummitJsrun
+    local_app_launcher: balsam.platform.app_run.LocalAppRun
+    mpirun_allows_node_packing: false # mpi_app_launcher supports multiple concurrent runs per node
+    serial_mode_prefetch_per_rank: 64 # How many jobs to prefetch from API in serial mode
+
+    # Pass-through parameters to mpirun when starting the serial mode launcher:
+    serial_mode_startup_params: 
+        cpu_affinity: none
+
+# ###################################################################
+# BALSAM Site modules
+# Any of the modules below can be disabled by setting the key: null
+#
+# For instance, `transfers: null` will disable the transfers module
+# if you are not performing any remote data transfers
+# ###################################################################
+
+# Scheduler service syncs the Balsam API with local queues
+scheduler:
+    scheduler_class: balsam.platform.scheduler.LsfScheduler
+    sync_period: 10 # Scheduler polling / API update interval in seconds
+
+    # Configure the queues to which BatchJobs may be submitted at this Site
+    allowed_queues:
+        batch:
+            max_nodes: 4608
+            max_queued_jobs: 100
+            max_walltime: 1440
+        
+
+    # Projects to which BatchJobs may be submitted:
+    allowed_projects:
+    - CSC388
+
+    # Supported "pass-through" parameters to the BatchJob template
+    # Provided as key:value pairs, where key is the parameter name
+    # and value is the *default* value of the parameter
+    optional_batch_job_params:
+        singularity_prime_cache: 'no'
```

Can remove this optional param from theta:

```suggestion
    optional_batch_job_params: {}
```